### ISSUE TITLE
Add multiple Themes to Deepin Terminal again

### ISSUE THREAD

**Issue by Y0ngg4n** [linuxdeepin/developer-center#1827]
It was so useful and handy to have the option to switch between multiple Themes in Deepin Terminal.
Please add this option again in Deepin Terminal.

**Comment by XMuli** [comment]
Black and light theme mode?

**Comment by Y0ngg4n** [comment]
@xmuli no. Deepin Terminal had more themes than black and light mode before update. If you made a right click you could select the terminal colours. It was a list of about 50 themes.

**Comment by XMuli** [comment]
Oh, I remember, I asked before, the theme of the new terminal is under development, and more and more themes will be supported later

**Comment by Y0ngg4n** [comment]
@xmuli Is there a roadmap for this? so you can view what the Progress is on this?

**Comment by XMuli** [comment]
I don’t. They are another team. I haven’t seen the road map (but someone made plans for them); we have complained about it internally before; I hope they will support multi-color themes earlier

**Comment by Y0ngg4n** [comment]
@xmuli ok thank you

**Comment by z966768** [comment]
The function of custom themes will be added later

**Comment by Y0ngg4n** [comment]
@z966768 it is not about custom themes. It is about all the themes that are removed with the update.

**Comment by z966768** [comment]
@Y0ngg4n I understand what you mean, similar to the theme selection function of the old version of the terminal, it will be added lat

**Comment by Y0ngg4n** [comment]
@z966768 ok thanks
